android/trifle/src/main/java/app/cash/trifle/TrifleApi.kt

```diff
@@ -13,6 +13,22 @@ object TrifleApi {
    */
   fun generateKeyHandle(alias: String): KeyHandle = KeyHandle.generateKeyHandle(alias)
 
+  /**
+   * Check if the [KeyHandle] exists and is valid.
```

Comment:
>   Check if the [KeyHandle] exists and is valid.

This reads a little confusing. it sounds like it checks whether it exists AND is valid.
when it really only checks whether it exists using `containsAlias`

we should probably document better to say keyhandle is valid only when it exists in the keystore. 
b/c my original question was when is keyhandle valid?

not sure if `valid` is the right word here?
it's probably clear to say `exists`, `doesExist`, `has`, `contains`?

Reply:
Thats fair. Was trying to keep the api and documentation consistent with iOS' implementation:
https://github.com/cashapp/trifle/blob/main/ios/Trifle/Sources/Trifle.swift#L49

however, could update the documentation better

Reply:
i think the ios doc should be updated too :p

Reply:
I'd do so in a separate PR